project/Build.scala

```diff
@@ -635,18 +624,21 @@ object Build {
       exportJars := true, // required so ScalaDoc linking works
 
       unmanagedSourceDirectories in Compile +=
-        baseDirectory.value.getParentFile / "shared/src/main/scala"
+        baseDirectory.value.getParentFile.getParentFile / "shared/src/main/scala"
   )
 
-  lazy val logging: Project = (project in file("logging/jvm")).settings(
+  lazy val logging: MultiScalaProject = MultiScalaProject(
+    id = "logging", base = file("logging/jvm")
```

Comment:
```suggestion
      id = "logging", base = file("logging/jvm")
```
There is the same thing several times in the file.